### DISCUSSION TITLE
Show Gold amount

### DISCUSSION THREAD

**Discussion by mirakhany** [isid0re/SoloLeveling#297] [Ideas]
HI. Your overlay is cool and very useful.
Would be nice if overlay shows gold amount.

**Comment by isid0re** [comment]
That's a nice suggestion. I'll see what I can do. Thanks

**Reply by mirakhany** [reply]
Thank you for show gold update.
It's cool! but please consider remove K, M for gold amount.
Some of country don't use K, M for numbers.
It's hard to fast recognize.

**Reply by isid0re** [reply]
No problem. I was concerned with space, but after testing it isnt affected. Will update to raw amount.
